gramps/gui/widgets/fanchart.py

```diff
@@ -1595,7 +1595,8 @@ def on_popup(self, obj, event, person_handle, family_handle=None):
                     continue
                 siblings.append(sib_id)
         # Collect a list of per-step-family step-siblings
-            for parent_h in [fam.get_father_handle(), fam.get_mother_handle()]:
+            for parent_h in [hnd for hnd in [fam.get_father_handle(),
```

Comment:
Instead of a list comprehension it maybe clearer to use:  `if not parent_h: continue`